gapic/src/main/com/google/gapid/GraphicsTraceView.java

```diff
@@ -103,6 +103,13 @@ public void store(TabArea.FolderInfo[] folders) {
     addListener(SWT.Dispose, e -> {
       models.follower.removeListener(this);
     });
+
+//    models.devices.addListener(new Devices.Listener() {
+//      @Override
+//      public void onReplayDevicesLoaded() {
+//        DeviceDialog.showSelectReplayDeviceDialog(getShell(), models, widgets);
+//      }
+//    });
```

Comment:
I suppose this part is not intended to be here?

Reply:
Indeed! removed.